redactionMap.js

```diff
@@ -57,7 +57,74 @@ const redactionMap = {
         childElementAttribute: 'src',
         value: '/favicon.ico'
       }
-    }
+    },
+    {
+      description: 'See connections (__+), this appears on profile pages top right corner',
```

Comment:
I don't think it makes sense to redact the person's connections -- while connections could be used to identify someone, I think they are more likely a useful indicator of how well connected an individual is, which is a meaningful signal.